src/components/AuthContainer.js

```diff
@@ -38,6 +39,25 @@ export default class AuthContainer extends Component {
       }, () => {
         console.warn('Error looking up user status')
       })
+
+      const [billingProjects, clusters] = await Promise.all([Rawls.listBillingProjects(), Leo.clustersList()])
+      let projectsWithoutClusters = _.difference(
```

Comment:
Were we going to do something with not assuming that we can use clusters without a certain label?

Reply:
That is tracked in #70.